src/common_fits.jl

```diff
@@ -206,7 +206,7 @@ function analyzeRB(ypts, seqlengths)
 
 	numRepeats = length(ypts)/length(seqlengths);
 
-	xpts = seqlengths[1 + floor((0:length(ypts)-1)./numRepeats)];
+	xpts = seqlengths[1 + floor(Int64,(0:length(ypts)-1)./numRepeats)];
```

Comment:
This should be fine. Alternatively, you can write this as
```julia
(0:length(ypts)-1) .÷ numRepeats
```
which is rather cute. You get that symbol with `\div<TAB>` at the REPL or in Atom\Sublime.